### PR TITLE
[REFACTORED RELATIONSHIP DOC]: bubbleset documentation

### PR DESCRIPTION
**Pull request by captain-Akshay**
**Notes for Reviewers**

This PR fixes #

Refactored relationships 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fe421e48e2ee2cd07769851aface6c6ba723714a at: https://657c45b503d8d20824c09f32--meshery-docs-preview.netlify.app

File: docs/pages/concepts/logical/relationships.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ---
 layout: default
 title: Relationships
-permalink: concepts/relationshps
+permalink: concepts/relationships
```
Comment (leecalcote) [inline docs/pages/concepts/logical/relationships.md:4]: Good catch, @captain-Akshay. Please run a full search across all docs to ensure that any links to this page are also updated. Especially, the toc.yaml.

File: docs/pages/concepts/logical/relationships.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 type: concepts
 abstract: "Meshery Relationships identify and facilitate genealogy between Components."
 language: en
 list: include
+image: /assets/img/bubbleset-example.gif
```
Comment (leecalcote) [inline docs/pages/concepts/logical/relationships.md:9]: Please find a more befitting directory.

File: docs/pages/concepts/logical/relationships.md [hunk 3]
```diff
@@ -23,56 +24,190 @@ The `isAnnotation` attribute of a Component determines whether the give Componen
...
+<details>
+    <summary>Frontend</summary>
+    <p>
+        MeshMap is an advanced and intuitively designed tool for creating and managing cloud-native infrastructure for Kubernetes. We leverage the Cytoscape library to enable the freestyle composition of your infrastructure. This involves drawing shapes, lines, text, and icons to visually represent the components and relationships within your Kubernetes clusters. Alternatively, you can quickly start by selecting a pattern from our catalog of predefined templates.
+    </p>
+    <div>
+        {% include alert.html type='dark' title='What is Cytoscape?' content='<a href="https://js.cytoscape.org/" target="_blank">Cytoscape</a> is a powerful and flexible JavaScript library for graph theory. It allows you to easily create interactive and customizable network graphs.' %}
+    </div>
+    <p>
+        In MeshMap, we provide a bottom dock for interacting with the relationships in the bubbleset. There are two main options for displaying relationships:
```
Comment (leecalcote) [inline docs/pages/concepts/logical/relationships.md:151]: Move this to docs.layer5.io

**Comment by l5io** [comment]
🚀 Preview for commit b43d4f70d812d8a220768a261a6d215f1b00bd99 at: https://657cac3fe9642c27229d8518--meshery-docs-preview.netlify.app

**Pull request closed without merging**